org.eclipse.xtext.tests/.classpath

```diff
@@ -6,7 +6,6 @@
 		</attributes>
 	</classpathentry>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.junit.JUNIT_CONTAINER/5"/>
```

Comment:
with the container tycho fails to resolve
without you have to select junit4 for the junit4 tests in run as dialogs. did not find a nice solution yet
see discussion in https://github.com/eclipse/xtext/issues/2197

Reply:
it is also about running single tests. currently in maintainance mode this is 50% of my usecases.
maybe there is also another option to make the vintage enginge visble to jdt.

Reply:
i dont know. 
the problem is we mix tests and a plugin.
and we need the imports for the plugin

Reply:
What do you mean by "we need the imports for the plugin"?

Reply:
xtext.testing has tests and is a real plugin

Reply:
Yes, I meant that: I seem to understand from the discussion that the JUnit container in the classpath should be enough (without import packages in the MANIFEST) to make everyone happy (running tests from Eclipse and Tycho).

Reply:
will it work if you use xtext.testing in mydsl.tests then?i assume no

Reply:
it is a plugin that is used in other plugins

Reply:
yes this is what i mean. you have to selct junit 4 in the dialog.

i have doubts a plugin can have code in osgi that is not on classpath. but i am not an osgi expert.
am also not sure how to properly test this.